installation/nginx.md

```diff
@@ -263,6 +297,10 @@ server {
 		proxy_set_header X-Real-IP              $remote_addr;
 		proxy_set_header X-Forwarded-For        $proxy_add_x_forwarded_for;
 		proxy_set_header X-Forwarded-Proto      $scheme;
+		satisfy                                 any;
+		allow                                   192.168.0.1/24;
+		allow                                   127.0.0.1;
+		deny                                    all;
```

Comment:
Ah I see, since the password file is included in that block, it'll still block outsiders unless they know the password, and won't do much against local security.